src/components/Web3ModalProvider.tsx

```diff
@@ -24,6 +24,9 @@ createWeb3Modal({
   projectId,
   featuredWalletIds: [BRAWE_WALLET_ID],
   includeWalletIds: [BRAWE_WALLET_ID],
+  themeVariables: {
+    '--w3m-accent': '#DD7143'
```

Comment:
You can also move this color to tailwind config under nextui plugin, and use it here in js, see [https://nextui.org/docs/customization/colors](https://nextui.org/docs/customization/colors)